versions.yml

```diff
@@ -24,4 +24,5 @@ allVersions:
   - "v2.18.0"
   - "v3.0.0"
   - "v3.1.0"
+  - "v3.2.0"
```

Comment:
I wouldn't add anything to the versions.yml until we are ready to release the 3.2 docs